src/main/scala/mesosphere/marathon/core/event/EventModule.scala

```diff
@@ -0,0 +1,93 @@
+package mesosphere.marathon.core.event
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import akka.event.EventStream
+import akka.pattern.ask
+import mesosphere.marathon.MarathonConf
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.election.ElectionService
+import mesosphere.marathon.core.event.impl.callback._
+import mesosphere.marathon.core.event.impl.stream._
+import mesosphere.marathon.metrics.Metrics
+import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer }
+import mesosphere.marathon.state.EntityStore
+import org.eclipse.jetty.servlets.EventSourceServlet
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.ExecutionContext
+
+class EventModule(
+    eventBus: EventStream,
+    actorSystem: ActorSystem,
+    conf: MarathonConf,
```

Comment:
It only reads conf.zkTimeoutDuration - it does not need the whole MarathonConf


> Am 18.07.2016 um 18:22 schrieb Gastón Kleiman <notifications@github.com>:
> 
> In src/main/scala/mesosphere/marathon/core/event/EventModule.scala <https://github.com/mesosphere/marathon/pull/4104#discussion_r71179493>:
> 
> > +import mesosphere.marathon.core.base.Clock
> > +import mesosphere.marathon.core.election.ElectionService
> > +import mesosphere.marathon.core.event.impl.callback._
> > +import mesosphere.marathon.core.event.impl.stream._
> > +import mesosphere.marathon.metrics.Metrics
> > +import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer }
> > +import mesosphere.marathon.state.EntityStore
> > +import org.eclipse.jetty.servlets.EventSourceServlet
> > +import org.slf4j.LoggerFactory
> > +
> > +import scala.concurrent.ExecutionContext
> > +
> > +class EventModule(
> > +    eventBus: EventStream,
> > +    actorSystem: ActorSystem,
> > +    conf: MarathonConf,
> HttpEventStreamServlet needs stuff from MarathonConf.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/mesosphere/marathon/pull/4104/files/2d3755c8a8c7e743ecdb026464f974daa07f70e2#r71179493>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAUK9eFcF86dRV_KH6CLio7Skn-EyxIFks5qW6hSgaJpZM4JNmzw>.
>